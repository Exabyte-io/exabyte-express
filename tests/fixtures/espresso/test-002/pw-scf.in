version https://git-lfs.github.com/spec/v1
oid sha256:6d960bed1fe33818a4a042db62bbd273a020454becd1be3406067fa9af869dc9
size 1136
