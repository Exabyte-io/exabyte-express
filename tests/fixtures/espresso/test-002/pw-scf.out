version https://git-lfs.github.com/spec/v1
oid sha256:a92f0a6f42827b60d0efbd9c04ea846e9281d0f14ce3d89769b54a3b99c45e34
size 11580
